e2e/execution_test.go

```diff
@@ -15,15 +15,16 @@ import (
 
 func testExecution(t *testing.T) {
 	var executionHash hash.Hash
-	var execPingCompleted *execution.Execution
+	var execPing *execution.Execution
+
 	ctx := metadata.NewOutgoingContext(context.Background(), passmd)
 	t.Run("stream", func(t *testing.T) {
-		t.Run("with nil filter", func(t *testing.T) {
+		t.Run("nil filter", func(t *testing.T) {
 			_, err := client.ExecutionClient.Stream(context.Background(), &pb.StreamExecutionRequest{})
 			require.NoError(t, err)
 		})
 		// TODO: no error are returned but it supposed to...
-		// t.Run("with not valid filter", func(t *testing.T) {
+		// t.Run("not valid filter", func(t *testing.T) {
```

Comment:
let's remove those test if the filter is not tested...
related to https://github.com/mesg-foundation/engine/pull/1463#discussion_r348532764